test/cli/test_cli_functions.sh

```diff
@@ -634,9 +634,61 @@ function account_cycle {
     account_reset_password "admin@noobaa.io"
     # testing nsfs accounts
     # account_nsfs_cycle TODO re-enable.
+    # update account default resource
+    account_update ""
+    # update crd account default resource
+    account_update "crd"
```

Comment:
we just want to check if crd variable is not empty so here we can leave it:
```suggestion
    # update account default resource
    account_update 
    # update crd account default resource
    account_update "crd"
```